odl/solvers/advanced/proximal_operators.py

```diff
@@ -846,6 +898,61 @@ def proximal_cconj_l1(space, lam=1, g=None, isotropic=False):
     prox_factory : function
         Factory for the proximal operator to be initialized
 
+    Notes
+    -----
+    The :math:`L_1`-norm :math:`F` is the functional
```

Comment:
distance
Also I prefer 1 as exponent up there, but that's a matter of taste. We should be consistent though.